pkg/websocket/websocket.go

```diff
@@ -1,7 +1,10 @@
 package websocket
 
+import "github.com/surrealdb/surrealdb.go/internal/rpc"
+
 type WebSocket interface {
 	Connect(url string) (WebSocket, error)
 	Send(method string, params []interface{}) (interface{}, error)
 	Close() error
+	LiveNotifications(id string) (chan rpc.RPCResponse, error)
```

Comment:
Same applies here as well,
people cannot import or use internal packages if we want to use we should move rpc package to pkg